src/v/net/server.cc

```diff
@@ -318,6 +325,30 @@ void server::setup_metrics() {
          sm::description(ssx::sformat("{}: Latency ", cfg.name)))});
 }
 
+void server::setup_public_metrics() {
+    namespace sm = ss::metrics;
+    if (!_proto) {
+        return;
+    }
+
+    std::string_view server_name(cfg.name);
+
+    if (server_name.ends_with("_rpc")) {
+        server_name.remove_suffix(4);
+    }
+
+    auto server_label = sm::label("server");
+
+    _public_metrics.add_group(
+      prometheus_sanitize::metrics_name("rpc:request"),
+      {sm::make_histogram(
+         "latency_seconds",
```

Comment:
nit: The other metric is labelled:
```c++
      "rpc",
      {sm::make_counter(
         "request_errors_total",
```
Which both have the same prefix. Is there a semantic reason to have these different?

Reply:
No, that's just how I was typing. I can change it.